python-client/giskard/models/model_explanation.py

```diff
@@ -31,13 +48,53 @@ def _get_background_example(df: pd.DataFrame, feature_types: Dict[str, str]) ->
 
 
 def _get_columns_original_order(prepared_dataset: Dataset, model: BaseModel, dataset: Dataset) -> list:
-    """Get columns of the prepared_dataset in the original order."""
+    """Return columns of the `prepared_dataset` in the original order.
+
+    Use `model` or `dataset` to infer columns' original order, which `model` was
+    trained with.
```

Comment:
Use `model` or `dataset` to deduce the original sequence of columns that the model was initially trained with.